pulp_docker/app/tasks/synchronize.py

```diff
@@ -39,8 +39,9 @@ def synchronize(remote_pk, repository_pk):
     repository = Repository.objects.get(pk=repository_pk)
     if not remote.url:
         raise ValueError(_('A remote must have a url specified to synchronize.'))
-    remove_duplicate_tags = [{'model': ManifestTag, 'field_names': ['name']},
-                             {'model': ManifestListTag, 'field_names': ['name']}]
+    # TODO this probably allows 2 "latest" tags for image manifests with different media types
+    # ie, v1/v2
+    remove_duplicate_tags = [{'model': ManifestTag, 'field_names': ['name', 'media_type']}]
```

Comment:
i think we should include just the name among the fields. fFr example: there can be just 1 tag with name 'latest' within the repo. It would whether point to a manifest or manifest list.

Reply:
@ipanova, do we have to give preferential treatment to manifest lists over manifests? If we enforce by name only, we would want the highest media type to be the one in the repo (I think).